src/pages/Wallet/WalletHeader/WalletHeader.js

```diff
@@ -0,0 +1,26 @@
+import WalletIcon from "@mui/icons-material/Wallet";
+import {
+  ContentContainer,
+  InnerCircle,
+  PendingTransfers,
+  Title,
+  TitleContainer,
+} from "./WalletHeaderStyled";
+
+const WalletHeader = ({ title }) => {
+  return (
+    <>
+      <ContentContainer>
+        <InnerCircle>
+          <WalletIcon style={{ color: "#fff", fontSize: 70 }} />
```

Comment:
I would move inline styles to WalletHeaderStyled.js as well.
Please also pull the latest version of the V1 branch into this one so I can merge it. V1 is our new main branch for now.